src/constants/ui.ts

```diff
@@ -1,9 +1,13 @@
 import { ERROR_CODES } from "../services/Errors";
 
-// Sensor data fallbacks for empty data fields
-const SENSOR_MESSAGE = {
+// Node data fallbacks for empty data fields
+const NODE_MESSSAGE = {
   NO_NAME: "(unnamed)",
   NO_LOCATION: "—",
+};
+
+// Sensor data fallbacks for empty data fields
+const SENSOR_MESSAGE = {
```

Comment:
Interesting. @m-mcgowan do you think we should split our messages into two separate objects or keep them all grouped under the newly renamed `NODE_MESSAGE`?